account-kit/react/src/tailwind/components/modal.ts

```diff
@@ -1,11 +1,8 @@
 import type { ComponentDef } from "../types";
 
 export const modalComponents: ComponentDef = {
-  ".modal": {
+  ".akui-modal": {
     "@apply radius-t-2 md:radius-2": {},
     "@apply bg-bg-surface-default": {},
-    ".modal-box": {
-      "@apply z-[1] px-6 py-4": {},
-    },
```

Comment:
is this not used?

Reply:
I was unable to find this classname in the codebase and when the modal renders in the dom.  I think it for something that was removed.